test/unit/rubric_criteria_test.rb

```diff
@@ -6,26 +6,28 @@
 require 'machinist'
 
 class RubricCriterionTest < ActiveSupport::TestCase
-  should validate_presence_of :assignment_id
-  should validate_numericality_of :assignment_id
-  should validate_numericality_of :weight
-  should validate_presence_of :weight
-
-  should validate_presence_of :rubric_criterion_name
-
-# FIXME there should be an easy "default" shoulda way to test the uniqueness
-# on the name on a specific scope
-
-#  context "a valid Rubric Criterion model" do
-#    setup do
-#      RubricCriterion.make
-#    end
-#
-#    should validate_uniqueness_of(
-#                :rubric_criterion_name).scope(:assignment_d)
-#  end
-
-  #Test that Criteria assigned to non-existant Assignment
+
+  context "A good rubric criterion model" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.